src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserRoleManagementResource.kt

```diff
@@ -260,4 +260,62 @@ class UserRoleManagementResource(
   ): UserRoleDetail {
     return userService.removeRoleFromUser(username, roleCode, caseloadId)
   }
+
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_ACCESS_ROLES_ADMIN') or hasRole('ROLE_MAINTAIN_ACCESS_ROLES')")
+  @PostMapping("/remove-roles/{roleCode}")
+  @Operation(
```

Comment:
Maybe it would be nice for this to be a DELETE but some frameworks do not support DELETEs with bodies thought the spec does allow it